Mage/src/main/java/mage/game/GameState.java

```diff
@@ -808,6 +808,25 @@ public boolean hasSimultaneousEvents() {
         return !simultaneousEvents.isEmpty();
     }
 
+    public void addSimultaneousLifeLoss(LifeLostEvent lifeLossEvent, Game game) {
```

Comment:
Current naming logic of all batch methods is wrong - it does not add real events to sim queue, only batches. So it must be renamed to addSimultaneousXxxEventToBatches - damaged, loseLife, tapped, etc.

Reply:
Going to rename just this function in this PR for now, unless youre good with renaming the rest in here.